src/lib/mina.ts

```diff
@@ -395,7 +395,7 @@ function LocalBlockchain({
           }
           events[addr][tokenId].push({
             events: p.body.events,
-            slot: networkState.globalSlotSinceHardFork.toString(),
+            slot: networkState.globalSlotSinceGenesis.toString(),
```

Comment:
@mitschabaude Since I removed `globalSlotSinceHardFork`, I replaced this with `globalSlotSinceGenesis`, would that be okay?